README.md

```diff
@@ -2,7 +2,9 @@
 
```

Comment:
We're also gonna need an image here on top, just like in the other Open Source packages :) Can you prepare that just as in the template? We will get an image for it when the designers are back!